beefy-primitives/tests/basic_lc_operations.rs

```diff
@@ -1,364 +0,0 @@
-// Copyright (C) 2020 Parity Technologies (UK) Ltd.
```

Comment:
I think these tests are worth keeping around. AFAIR they were not part of the Substrate PR, so they should stay here.

Reply:
Yep, I'm thinking about adding a `light-client` folder and putting the tests there.

Reply:
Yup, that makes a lot of sense. My initial idea was that the `light-client` test stuff can actually evolve into a proper light client implementation that we will be able to use on-chain eventually for bridges.